packages/medusa/src/api/routes/admin/orders/create-swap.ts

```diff
@@ -247,3 +242,83 @@ export default async (req, res) => {
 
   res.status(idempotencyKey.response_code).json(idempotencyKey.response_body)
 }
+
+export class AdminPostOrdersOrderSwapsReq {
+  @IsArray()
+  @IsNotEmpty()
+  @ValidateNested({ each: true })
+  @Type(() => ReturnItem)
+  return_items?: ReturnItem[]
```

Comment:
When applying `@IsNotEmpty()` I assume it is wrong to define the variable as `return_items?`